code/modules/mob/mob.dm

```diff
@@ -215,7 +215,7 @@
  * set del_on_fail to have it delete W if it fails to equip
  * unset redraw_mob to prevent the mob from being redrawn at the end.
  */
-/mob/proc/equip_to_slot_if_possible(obj/item/W, slot, ignore_delay = TRUE, del_on_fail = FALSE, warning = TRUE, redraw_mob = TRUE, permanent = FALSE, override_nodrop = FALSE)
```

Comment:
`permanent` was unused, and this proc really should not handle adding NODROP anyway